package.json

```diff
@@ -34,18 +34,21 @@
       ]
     }
   },
+  "peerDependencies": {
+    "react": "^0.13.0 || ^0.14.0 || >=15"
```

Comment:
How have you decided on compatibility with this specific version range?

Reply:
@aknuds1 Since I think this package only uses `React.createElement` which is a core API, in theory it should work with most React versions.

This version range is set to be inclusive for versions of React since 2015 (with React becoming stable in 2016 going from `v0.14` to `v15`).  This will avoid issues if anyone happens to be using an old version of React (while unlikely).  

If it makes sense to support something more modern, we can do `"react": ">=15"` or `"react": ">=16"`.

Reply:
Sounds good, thanks for the explanation @woofers :)